hdfsee.py

```diff
@@ -0,0 +1,283 @@
+from widget import My_slider, Radio
+from bttn_peak import Bttn_peak
+
+
+# Creating arguments for parsing.
+
+class Image():
+    """
+    Class for the main image.
+    fig - object class matplotlib.figure:
+          here subplots and widgets will be added.
+    ax - matplotlib.pyplot.subplot for displaying image and updating with
+         the widget.
+    vmax - for contrast.
+    vmin - for contrast.
+    cmap - colour mappings.
+    matrix - Array filled with data from the panels.
+    image - returned by matplotlib imshow.
+    peaks - list containing peak objects from 'peak_h5' module.
+    bad_places - list containing 'bad places' from 'panel' module.
+    axis_list - list of matplotlib.pyplot.axes objects for locations of the
+                buttons.
+    list_active_peak - Flags list showing which type of peaks were selected.
+    """
+    PARSER = argparse.ArgumentParser()
+    PARSER.add_argument('filename', nargs=1, metavar="name.H5",
+                        help='Display this image.')
+    PARSER.add_argument('-g', "--geomfile", nargs=1, metavar='name.GEOM',
+                        help='Use geometry from file' +
+                        ' to display arrnagmrnt panels')
+    PARSER.add_argument('-p', '--peaks', nargs=1, metavar='name.STREAM',
+                        help='use to display peaks' +
+                        ' from stream is used only witch geom')
+    # Parsing command line arguments.
+    ARGS = PARSER.parse_args()
+    # Variable for running mode.
+    WHICH_ARGUMNENT_IS_USED = {'display_only_file': False,
+                               'display_arrangment_view': False,
+                               'dispaly_with_peaks': False}
+    # Variable for filename.
+    FILE_H5_NAME = ARGS.filename[0]
+    if ARGS.geomfile:
+        # Check if the geometry file was provided.
+        FILE_GEOM_NAME = ARGS.geomfile[0]
+        if ARGS.peaks:
+            FILE_STREAM_NAME = ARGS.peaks[0]
+            WHICH_ARGUMNENT_IS_USED['dispaly_with_peaks'] = True
+            WHICH_ARGUMNENT_IS_USED['display_arrangment_view'] = False
+        else:
+            # Only the geometry file was provided.
+            FILE_STREAM_NAME = None
+            WHICH_ARGUMNENT_IS_USED['display_arrangment_view'] = True
+    # Image file without geometry.
+    else:
+        if ARGS.peaks:
+            print('Displaying panels without geometry reconstruction.')
+        FILE_STREAM_NAME = None
+        FILE_GEOM_NAME = None
+        WHICH_ARGUMNENT_IS_USED['display_only_file'] = True
+
+    try:
+        GEOM = c.load_crystfel_geometry(FILE_GEOM_NAME)
+        # Dictionary with information about the image: panels, bad places.
+        # Tuple for the minimal images size.
+        SIZE_OBRAZ = g.compute_min_array_size(g.compute_pix_maps(GEOM))
+    except FileNotFoundError:
+        print("Error while opening geometry file.")
+        sys.exit()
+    except TypeError:
+        # No geometry file was provided.
+        SIZE_OBRAZ = None
```

Comment:
`ALL_CAPITAL` variables are for [constants](https://www.python.org/dev/peps/pep-0008/#constants). all variables should contains [lowercase and underscore](https://www.python.org/dev/peps/pep-0008/#function-and-variable-names)

Reply:
Addresed in: `7ec89d9b88a8d30f12850862f31c0798c5bef52a`